apps/frontend/components/Filters/DateRangeFilter.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box } from '@mui/material';
+import { DatePicker } from '@mui/x-date-pickers';
+import dayjs, { Dayjs } from 'dayjs';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+export interface DateRange {
+  startDate: Dayjs;
+  endDate: Dayjs;
+}
+
+interface IProps {
+  value: DateRange;
+  onChange: (range: DateRange) => void;
+}
+
+export const DateRangeFilter = ({ value, onChange }: IProps): JSX.Element => {
+  const intl = useIntl();
+
+  return (
+    <Box display="flex" justifyContent="center" mb={3}>
+      <DatePicker
+        label={intl.formatMessage({
+          id: 'validation_search_params.start_date',
+        })}
+        maxDate={dayjs(new Date())}
+        value={value.startDate}
+        onChange={newStartValue => {
+          if ((newStartValue as Dayjs) > value.endDate) {
+            onChange({
```

Comment:
im not sure abt doing it this way, i think a react hook form validation would be better but per our discussion this is also dependent on the UX we are aiming for